disperser/common/blobstore/shared_storage.go

```diff
@@ -78,22 +80,31 @@ func NewSharedStorage(bucketName string, s3Client s3.Client, blobMetadataStore *
 func (s *SharedBlobStore) StoreBlob(ctx context.Context, blob *core.Blob, requestedAt uint64) (disperser.BlobKey, error) {
```

Comment:
same with errors in this file

Reply:
See my comment in the other thread: https://github.com/Layr-Labs/eigenda/pull/823#discussion_r1812887627